bindings/python/pymongocrypt/mongocrypt.py

```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from bson import encode as bson_encode
```

Comment:
I'm a little wary of directly importing pymongo in pymongocrypt because it creates a circular dependency. That's why I used dependency injection for bson.encode in the `MongoCryptCallback` class. 

For example we can remove this import by passing the callback (or just the encode method) to MongoCrypt:
```
MongoCrypt(mongo_crypt_opts, callback=callback)
# or
MongoCrypt(mongo_crypt_opts, bson_encode=callback.bson_encode)
```